Venmo/src/main/java/com/braintreepayments/api/venmo/VenmoRequest.kt

```diff
@@ -0,0 +1,123 @@
+ * profile, and it will show up in the Venmo app as a "Connected Merchant".
+ * Venmo profile IDs can be found in the Braintree Control Panel. Leaving this
+ * null` will use the default Venmo profile.
+ *
+ * @property displayName Optional - The business name that will be displayed in the Venmo app
+ * payment approval screen. Only used by merchants onboarded as PayFast channel partners.
+ *
+ *
+ * @property collectCustomerShippingAddress Optional - The boolean value of the flag that
+ * signifies whether customer shipping address will be collected. Whether or not shipping address
+ * should be collected and displayed on Venmo pay sheet.
+ *
+ * @property collectCustomerBillingAddress Optional - The boolean value of the flag that
+ * signifies whether customer billing address will be collected. Whether or not billing address
+ * should be collected and displayed on Venmo pay sheet.
+ *
+ * @property totalAmount Optional - The grand total amount of the transaction that will be
+ * displayed on the pay sheet. The grand total amount of the transaction that will be displayed
+ * on the pay sheet.
+ *
+ * @property subTotalAmount Optional - The subtotal amount of the transaction, excluding taxes,
+ * discounts, and shipping. The subtotal amount of the transaction, excluding taxes, discounts,
+ * and shipping.
+ *
+ * @property discountAmount Optional - The total discount amount applied on the transaction. The
+ * total discount amount applied on the transaction. If this value is set, `totalAmount` must
+ * also be set.
+ *
+ * @property taxAmount Optional - The total tax amount applied to the transaction. The total tax
+ * amount applied to the transaction. If this value is set, `totalAmount` must also be set.
+ *
+ * @property shippingAmount Optional - he shipping amount charged for the transaction. The
+ * shipping amount charged for the transaction. If this value is set, `totalAmount` must also be
+ * set.
+ *
+ * @property isFinalAmount Optional - The boolean value of the flag that signifies whether the purchase
+ * amount is the final amount. Indicates whether the purchase amount is the final amount.
+ * Defaults to false.
+ *
+ * @property isFinalAmountAsString - Whether or not the purchase amount is the final amount as a
+ * string value.
+ */
+@Parcelize
+class VenmoRequest @JvmOverloads constructor(
+    @VenmoPaymentMethodUsage val paymentMethodUsage: Int,
+    var lineItems: ArrayList<VenmoLineItem> = ArrayList(),
+    var shouldVault: Boolean = false,
+    var profileId: String? = null,
+    var displayName: String? = null,
+    var collectCustomerShippingAddress: Boolean = false,
+    var collectCustomerBillingAddress: Boolean = false,
+    var totalAmount: String? = null,
+    var subTotalAmount: String? = null,
+    var discountAmount: String? = null,
+    var taxAmount: String? = null,
+    var shippingAmount: String? = null,
+    var isFinalAmount: Boolean = false
+) : Parcelable {
+
+    val paymentMethodUsageAsString: String?
```

Comment:
If we convert `VenmoPaymentMethodUsage` to an enum, we should be able to use `SINGLE_USE`.name and `MULTI_USE`.name to get the string value of the enum.

Reply:
Done